cogs/mhelp.py

```diff
@@ -15,7 +15,7 @@ async def mhelp(self, ctx):
 
 Supported tags:
 ```
-    -t, --transparent, -i, --save_as_gif, -s, --save_last_frame
+    -i (--save_as_gif), -s (--save_last_frame), -t (--transparent), --renderer=opengl, --use_projection_fill_shaders, --use_projection_stroke_shaders
```

Comment:
```suggestion
    -i (--format=gif), -s (--save_last_frame), -t (--transparent), --renderer=opengl, --use_projection_fill_shaders or --use_projection_stroke_shaders.
```
`save_as_gif` is deprecated